index.html

```diff
@@ -44,7 +44,149 @@ <h5> I can help you build a product ,
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+         <div class="profesional p-3">
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+         <div class="profesional p-4">
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+         <div class="profesional p-5">
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+         <div class="profesional p-6">
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
```

Comment:
- [ ] The button (e.g., "See project") should follow the button styling: Active, Hover, and Pressed (see design below), and please implement it
### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/193013682-4b947d62-c543-4a72-bcba-1bd52f9f5e5b.png)